packages/design-system/src/theme.ts

```diff
@@ -1,13 +1,47 @@
-import { theme as chakraTheme } from '@chakra-ui/core';
+import { DefaultTheme, theme as chakraTheme } from '@chakra-ui/core';
 
-export const theme: typeof chakraTheme = {
+interface MetaColors {
+  dark: string;
+  purpleBoxDark: string;
+  purpleBoxLight: string;
+  purpleTag: string;
+  blueLight: string;
+  cyanText: string;
+}
+
+interface MetaTheme {
+  colors: MetaColors;
+}
+
+type Theme = DefaultTheme & MetaTheme;
+
+export const theme: Theme = {
   ...chakraTheme,
   colors: {
     ...chakraTheme.colors,
+    // metagame colors
+    purple: {
+      50: '#f1e7ff',
+      100: '#cfbafc',
+      200: '#ab8df3',
+      300: '#8460ed',
+      400: '#5a32e6',
+      500: '#4d19cd',
+      600: '#4412a0',
+      700: '#380d74',
+      800: '#260747',
+      900: '#11021d',
+    },
+    dark: '#1B0D2A',
+    purpleBoxDark: '#261943',
+    purpleBoxLight: '#392373',
+    purpleTag: '#40347C',
+    blueLight: '#A5B9F6',
+    cyanText: '#79F8FB',
```

Comment:
Similarly with these, we should use colors out of the color palette. If the color palette doesn't have the colors we want, we should add them there. In fact, we can just generate the whole purple swatch based on the shade of purple we have by using something like this: https://smart-swatch.netlify.app/